src/app/master-view/master-view.component.html

```diff
@@ -91,7 +88,9 @@ <h5 class="content">
         </div>
       </div>
       <div class="column-layout group_18">
-        <igx-grid [data]="northwindJasonApiCustomerOrdersCustomerId" primaryKey="orderID" displayDensity="cosy" rowSelection="single" [hideRowSelectors]="true" [allowFiltering]="true" filterMode="excelStyleFilter" class="grid">
+        <igx-grid [data]="northwindJasonApiCustomerOrdersCustomerId" primaryKey="orderID" displayDensity="cosy" rowSelection="single" [hideRowSelectors]="true" [allowFiltering]="true" filterMode="excelStyleFilter" class="grid"
+          (selected)="orderSelected($event.cell.row.data.orderID)"
```

Comment:
Not sure whether this is the right event for the grid to simulate what we are trying. @pmoleri mentioned something was not correct in the docs. But can't find the link now.